### DISCUSSION TITLE
Cannot import handle from hono/aws-lambda

### DISCUSSION THREAD

**Discussion by Amen-Musingarimi** [honojs/hono#3188] [Q&A]
Hello @everyone, I have created my AWS Gateway API, which is linked to the Lambda function created with Hono.  In that Hono function I am using that 'handle' function imported from 'hono/aws-lambda' to handle the hono app. However, when I test the API, I get an error that say, TypeError: handle  is not  a function.  What ioos it  that I might be doing wrong here? Please help
 

![Screenshot (362).png](https://github.com/user-attachments/assets/820774b5-d0e5-49f1-ba91-e0f7651217cd)

![Screenshot (363).png](https://github.com/user-attachments/assets/01ccd57e-4d1b-48c8-ac49-51d3e1ba2791)

![Screenshot (361).png](https://github.com/user-attachments/assets/fea2675a-20dd-47f2-a983-7843d2df8273)

**Comment by Amen-Musingarimi** [comment]
@yusukebe Can  you help?

**Reply by yusukebe** [reply]
@Amen-Musingarimi 

Try it:

```ts
const { handle } = require('hono/aws-lambda')
```

**Reply by Amen-Musingarimi** [reply]
Thank you it worked. When linking an AWS backend environment,  what function do you use in JavaScript,  i realised the 'import API from aws-amplify/api is no longer working?
